src/xcode/ENA/ENA/Source/Extensions/String+Insertion.swift

```diff
@@ -0,0 +1,30 @@
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import UIKit
+
+extension String {
+
+	func inserting(string: String, style: ENAFont) -> NSAttributedString {
```

Comment:
🤔`inserting` would suggest that the string only gets something added, but some formatting seems to happen here as well?

Reply:
Do you have a name suggestion?

Reply:
```suggestion
	func inserting(_ string: String, as: ENAFont) -> NSAttributedString {
```
Then the call side would look like this:
`AppStrings.ExposureSubmissionDispatch.hotlineButtonDescription.inserting(AppStrings.ExposureSubmissionDispatch.positiveWord, as: .headline)`

Reply:
Is the swift keyword 'as' usable?

Reply:
Damn you are right, hmm.
But do you agree that this function deserves a better naming our not? :)

Reply:
We narrowed down the functionality now and renamed it to `inserting(emphasizedString:)`. We think that makes it much clearer. Do you agree?

Reply:
Lovely 💫